functions/Invoke-DbaAdvancedUpdate.ps1

```diff
@@ -40,6 +40,9 @@ Function Invoke-DbaAdvancedUpdate {
         This avoids overwhelming you with "sea of red" exceptions, but is inconvenient because it basically disables advanced scripting.
         Using this switch turns this "nice by default" feature off and enables you to catch exceptions with your own try/catch.
 
+    .PARAMETER ExtractPath
+        Lets you specify a location to extract the update file to on the system requiring the update. e.g. C:\temp
+
     .EXAMPLE
     PS C:\> Invoke-DbaAdvancedUpdate -ComputerName SQL1 -Action $actions
```

Comment:
An example is still needed for this command. It is not an internal only command.

Reply:
So it seems to get called from Update-DbaInstance using an action list, I am really not sure how to write the example for that. Perhaps someone could help me out.

There is currently only 1 example which assumes a list of actions has been generated somewhere else with no explanation how.

Also synopsis says
"Designed for internal use"

Reply:
This function is not located in the internal directory and is exported as a public function. It can say designed or whatever else it wants to in the CBH, it still remains as a public function and therefore requires an example for the parameter. I don't know how to spell it any different.

Reply:
I have added the example and also switched up the order of the help (not required necessarily, but works with my ocd ;)) Check out more info here: https://github.com/sqlcollaborative/dbatools/pull/5529/commits/d247567b09d756b8b7441605bc0eacaa9d58af73

Reply:
Thanks @cl I'll have a look.